councilmatic/settings_deployment.py.example

```diff
@@ -6,8 +6,8 @@ import os
 SECRET_KEY = 'replacethiswithsomethingsecret'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-# Set this to True while you are developing
-DEBUG = False
+# Set this to False while you are in production
+DEBUG = True
```

Comment:
fyi, this also resolves 404s on all images and js/css, and workarounds like this:
https://docs.djangoproject.com/en/1.9/howto/static-files/#serving-static-files-during-development